rustls/Cargo.toml

```diff
@@ -17,6 +17,35 @@ build = "build.rs"
 [features]
 default = []
 
+# XXX TODO UPDATE NOTE(S) HERE
+# NOTE: This unstable & undocumented feature is intended to help make it easier to configure
+# docs.rs to generate the documentation for all features where this is needed,
+# excluding any potentially problematic features such as:
+# - `unsafe-assume-single-core`
+# - FIPS (if added back to this fork someday in the future)
+#
+# XXX TODO UPDATE NOTE(S) HERE
+# IMPORTANT: KEEP THIS FEATURE NAME IN SYNC with the following:
+# - [package.metadata.docs.rs] section further below
+# - docs generate job configured in: .github/workflows/docs.yml
+# XXX TODO RENAME
+# XXX TBD UPDATE CI to use `--features unstable-...`  INSTEAD OF USING all-features-except - ??? ???
+unstable-std-extra = [
```

Comment:
reconsider this feature name; I am thinking of something like "unstable-api-all" may be a bit clearer & easier to understand